numpy/doc/broadcasting.py

```diff
@@ -124,7 +125,7 @@
  (5,)
 
  >>> x + y
- <type 'exceptions.ValueError'>: shape mismatch: objects cannot be broadcast to a single shape
+ <type 'exceptions.ValueError'>: operands could not be broadcast together with shapes (4,) (5,)
```

Comment:
Can you replace `<type 'exceptions.ValueError'>` with `ValueError`?

Reply:
Done!